pkg/restore-util/client.go

```diff
@@ -181,3 +195,84 @@ func (c *pdClient) ScanRegions(ctx context.Context, key, endKey []byte, limit in
 	}
 	return regionInfos, nil
 }
+
+// LabelConstraint is used to filter store when trying to place peer of a region.
+type LabelConstraint struct {
```

Comment:
Why not use `"github.com/pingcap/pd/server/schedule/placement". LabelConstraint `?